From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] debian/qs_eval_globalize

Disable qs_eval code which tries to load locally built plugins.

Signed-off-by: Modestas Vainius <modestas@vainius.eu>

---
 qtbindings/qs_eval/main.cpp |   12 ++++++------
 1 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/qtbindings/qs_eval/main.cpp b/qtbindings/qs_eval/main.cpp
index eec0395..f046030 100644
--- a/qtbindings/qs_eval/main.cpp
+++ b/qtbindings/qs_eval/main.cpp
@@ -124,6 +124,7 @@ int main(int argc, char *argv[])
     else
         app = new QApplication(argc, argv);
 
+#if 0
     QDir dir(QApplication::applicationDirPath());
     if (dir.dirName() == QLatin1String("debug") || dir.dirName() == QLatin1String("release"))
         dir.cdUp();
@@ -136,6 +137,7 @@ int main(int argc, char *argv[])
     QStringList paths = app->libraryPaths();
     paths <<  dir.absolutePath();
     app->setLibraryPaths(paths);
+#endif
 
     QScriptEngine *eng = new QScriptEngine();
 
@@ -159,17 +161,15 @@ int main(int argc, char *argv[])
     if (!failExtensions.isEmpty()) {
         if (failExtensions.size() == extensions.size()) {
             qWarning("Failed to import Qt bindings!\n"
-                     "Plugins directory searched: %s/script\n"
-                     "Make sure that the bindings have been built, "
+                     "Make sure that the bindings have been installed, "
                      "and that this executable and the plugins are "
-                     "using compatible Qt libraries.", qPrintable(dir.absolutePath()));
+                     "using compatible Qt libraries.");
         } else {
             qWarning("Failed to import some Qt bindings: %s\n"
-                     "Plugins directory searched: %s/script\n"
-                     "Make sure that the bindings have been built, "
+                     "Make sure that the bindings have been installed, "
                      "and that this executable and the plugins are "
                      "using compatible Qt libraries.",
-                     qPrintable(failExtensions.join(", ")), qPrintable(dir.absolutePath()));
+                     qPrintable(failExtensions.join(", ")));
         }
     }
 
-- 
tg: (95a3b52..) debian/qs_eval_globalize (depends on: upstream)
